src/pages/dar_application/ResearcherInfo.js

```diff
@@ -232,6 +238,28 @@ export default function ResearcherInfo(props) {
             ])
           ])
         ]),
+        div({className: 'row no-margin'}, [
```

Comment:
Doesn't change much in terms of appearance but is there any reason you chose not to wrap this in a `div({ className: 'form-group' }, [...])` block like the other questions?